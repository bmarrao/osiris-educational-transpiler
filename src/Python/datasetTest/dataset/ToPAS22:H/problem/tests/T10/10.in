5
10 45
RW IMU LRN ej DLDCn lDcM CDG KRQJo jIcn H
IMU LRN
lDcM ej
H LRN
jIcn LRN
KRQJo lDcM
RW IMU
DLDCn jIcn
CDG RW
CDG jIcn
IMU jIcn
DLDCn ej
LRN ej
DLDCn IMU
LRN DLDCn
RW LRN
CDG KRQJo
ej IMU
H KRQJo
jIcn lDcM
ej RW
CDG H
H IMU
KRQJo IMU
jIcn RW
RW H
H ej
H DLDCn
jIcn KRQJo
lDcM H
CDG LRN
LRN KRQJo
DLDCn CDG
DLDCn RW
jIcn ej
KRQJo ej
lDcM IMU
jIcn H
CDG IMU
LRN lDcM
KRQJo DLDCn
RW lDcM
lDcM DLDCn
lDcM CDG
CDG ej
KRQJo RW
10 9
ulKYG XYYRZ U XLkr krVgv j q Wh UKt xFOVw
krVgv U
ulKYG j
XLkr q
XLkr XYYRZ
xFOVw Wh
ulKYG krVgv
Wh UKt
xFOVw ulKYG
XLkr xFOVw
10 10
SH ulH xEudJ owh eA Mo m uV fma DUcec
eA m
ulH DUcec
xEudJ uV
uV fma
uV DUcec
uV SH
Mo m
fma m
owh eA
xEudJ SH
10 11
GlY tbZo Cxk hicL C Fze wVkPz UF Qem hf
wVkPz UF
Fze Cxk
hf Qem
Fze UF
GlY Fze
Cxk hicL
hicL hf
wVkPz hf
GlY UF
wVkPz tbZo
C GlY
10 12
xQx g O Uxfq M TRdVP yz Tws LapU q
yz xQx
q xQx
Tws O
xQx M
O g
q O
g M
yz g
xQx Uxfq
Uxfq Tws
q M
LapU O
