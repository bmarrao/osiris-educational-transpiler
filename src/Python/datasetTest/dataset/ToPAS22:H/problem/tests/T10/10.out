Impossivel
Possivel: 1178880
Possivel: 1036800
Possivel: 913920
Possivel: 733440
